### PR TITLE
Update parameters/tolerances based on precision

### PR DESCRIPTION
**Pull request by sriharshakandala**
Update parameters/tolerances based on precision

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by tapios** [comment]
@sriharshakandala I left a few additional comments on the other (merged PR). Could you incorporate them here and try out how it affects results?

**Comment by sriharshakandala** [comment]
> @sriharshakandala I left a few additional comments on the other (merged PR). Could you incorporate them here and try out how it affects results?

Sure!

File: src/rte/RTESolverKernels.jl [hunk 1]
```diff
@@ -166,7 +166,7 @@ function rte_lw_2stream_source!(
     #k_min = FT === Float64 ? FT(1e-12) : FT(1e-4)
     k_min = FT(1e4 * eps(FT))
     lw_diff_sec = FT(1.66)
-    τ_thresh = sqrt(eps(FT))
+    τ_thresh = eps(FT)^(1 / 4)
```
Comment (simonbyrne) [inline src/rte/RTESolverKernels.jl:169]:
```suggestion
    τ_thresh = eps(FT)^FT(1 / 4)
```
since `1/4` will promote to `Float64`

**Comment by sriharshakandala** [comment]
Replaced by https://github.com/CliMA/RRTMGP.jl/pull/421

**Pull request closed without merging**